django_filters/rest_framework/backends.py

```diff
@@ -113,6 +113,26 @@ def get_coreschema_field(self, field):
             description=str(field.extra.get('help_text', ''))
         )
 
+    def build_coreapi_field(self, name, field):
+        return compat.coreapi.Field(
+            name=name,
+            required=field.extra['required'],
+            location='query',
+            schema=self.get_coreschema_field(field),
+        )
+
+    def get_schema_field_names(self, field_name, field):
+        """
+        Get the corresponding schema field names required to generate the openAPI schema
+        by referencing the widget suffixes if available.
+        """
+        try:
+            suffixes = field.field_class.widget.suffixes
+        except AttributeError:
+            return [field_name]
+        else:
+            return [field_name] if not suffixes else [f'{field_name}_{suffix}' for suffix in suffixes if suffix]
```

Comment:
I added this ternary in case an empty ```suffix``` list is declared. I saw this at ```SuffixedMultiWidget``` but I wasn't sure if that was only ever subclassed or used in it's base format